metricbeat/module/docker/docker.go

```diff
@@ -50,6 +50,8 @@ func FetchStats(client *docker.Client) ([]DockerStat, error) {
 
 	containersList := []DockerStat{}
 	for _, container := range containers {
+		// This is currently very inefficient as docker calculates the average for each request,
+		// means each request will take at least 1s. Getting all stats at once is implemented here: https://github.com/docker/docker/pull/25361
 		containersList = append(containersList, exportContainerStats(client, &container))
```

Comment:
Any idea how long each request takes? Does each request block for some time period so that it can take two samples in order to calculate a delta or percentage?